backend/src/modules/doubles-tournament/core/domain/TeamId.ts

```diff
@@ -5,7 +5,7 @@ export class TeamId {
 
   static from(teamId: string): TeamId {
     if (teamId.length <= 0) {
-      throw new Error('TeamId cannot be empty!');
+      throw new Error('TournamentTeamId cannot be empty!');
```

Comment:
czemu zmieniłeś nazwę w module doubles-tournament?

Reply:
Przepraszam, pomyłka, :(
Postaram się to cofnąć